### ISSUE TITLE
Error uploading Uruguay test data

### ISSUE THREAD

**Issue by VictoriaVlad** [openspending/openspending#1151]
Had an upload session to upload some test data with the pilot partner in Uruguay and got this error when trying to publish the data:

![uruguay test file error](https://cloud.githubusercontent.com/assets/17012538/20063609/bbc4eea6-a4bc-11e6-89b1-69a6490b8239.jpeg)

The csv file is available here
https://drive.google.com/open?id=0B4mVvh--ffocSWZDandvUFBGU1k
@DianaMPG

**Comment by akariv** [comment]
Can you also attach the datapackage.json file as well?

**Comment by VictoriaVlad** [comment]
@akariv, the .json file is available here https://drive.google.com/open?id=0B4mVvh--ffocSVZoZDRxdkdHZFU

**Comment by cyberbikepunk** [comment]
I don't have access to the csv and json but it looks like a dimension is trying to be parsed as an amount

**Comment by VictoriaVlad** [comment]
@cyberbikepunk I gave you access just now

**Comment by cyberbikepunk** [comment]
@VictoriaVlad it's not what i thought 

@akariv am i correct to assume that `description` columns are missing a `descriptionfor` property akin to the `labelfor`?

**Comment by akariv** [comment]
I think this is a problem with `tabulator` failing to parse a seemingly valid file.

@roll, can you take a look:
```python
# import tabulator, version 0.10.4
>>> import tabulator

# read the offending row using topen - see the field is spread over many columns
>>> list(iter(tabulator.topen('/Users/adam/Downloads/uy_presupuesto_2015_2019.csv').open()))[71][10:15]
/Users/adam/.pyenv/versions/3.5.2/lib/python3.5/site-packages/tabulator/topen.py:108: UserWarning: Function "topen" is deprecated [v0.6-v1)
  warnings.warn(message, UserWarning)
['Formación de profesionales de la educación, incluyendo la formación de educadores sociales. La Ley General de Educación Nº 18.43, en su artículo nro. 31 define la formación en educación. Ésta... \'"se concebirá como enseñanza terciaria universitaria y abarcará la formación de maestros', ' maestros técnicos', ' profesores', ' profesores de educación física y educadores sociales', ' así como de otras formaciones que el Sistema Nacional de Educación requiera\'""."']

# Just as a reference, open the raw file and read that line as is.
>>> repr(open('/Users/adam/Downloads/uy_presupuesto_2015_2019.csv').readlines()[71])
'\'"A.N.E.P.","Administración Nacional de Educación Pública","CFed","Consejo de Formación en Educación",2015,8,"EDUCACIÓN","Comprende los servicios prestados a alumnos y estudiantes, y los servicios docentes; la formulación y administración de políticas educativas, aplicación de normas, regulación, autorización y supervisión de los centros de enseñanza, así como la investigación aplicada y el desarrollo experimental en relación con los asuntos y servicios docentes; los gastos relacionados con la administración y gestión de centros educativos que imparten educación preescolar, primaria, secundaria y universitaria, así como otras instituciones que brindan formación y capacitación. Comprende la prestación de servicios auxiliares de la educación, como transporte, seguridad, alimentación, alojamiento, atención médica y odontológica y otros servicios auxiliares conexos. Comprende además las escuelas y academias militares, las escuelas de policía que imparten enseñanza general y la formación para el servicio exterior.",5,"Formación en Educación","Formación de profesionales de la educación, incluyendo la formación de educadores sociales. La Ley General de Educación Nº 18.43, en su artículo nro. 31 define la formación en educación. Ésta... \\\'""se concebirá como enseñanza terciaria universitaria y abarcará la formación de maestros, maestros técnicos, profesores, profesores de educación física y educadores sociales, así como de otras formaciones que el Sistema Nacional de Educación requiera\\\'"".",25,5,25005,117396841,"Funcionamiento",".007937"\\n\''
```

**Comment by roll** [comment]
@akariv 
Fix's just waiting your review - https://github.com/frictionlessdata/tabulator-py/pull/131

**Comment by DianaMPG** [comment]
thanks @akariv and @roll

**Comment by VictoriaVlad** [comment]
@roll how can I simply explain to the partner about what the error was? thank you kindly

**Comment by roll** [comment]
@VictoriaVlad 
It was a program error in underlying libraries (`tabulator` version update is required to fix). Parser was in trouble with this cell because of double quotes - `"Formación de profesionales de la educación, incluyendo la formación de educadores sociales. La Ley General de Educación Nº 18.43, en su artículo nro. 31 define la formación en educación. Ésta... '""se concebirá como enseñanza terciaria universitaria y abarcará la formación de maestros, maestros técnicos, profesores, profesores de educación física y educadores sociales, así como de otras formaciones que el Sistema Nacional de Educación requiera'""."`

**Comment by DianaMPG** [comment]
@roll thank you for this. So in layman terms, there was nothing wrong with the dataset? If so, we could give this advice to our project partners when submitting their data.

**Comment by pwalsh** [comment]
"A programming error due to the complexities of CSV and Excel files. The dataset was fine, it just raised a bug in our system that has now been fixed."
